sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ObjectExpressionsSuite.scala

```diff
@@ -641,6 +641,18 @@ class ObjectExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkObjectExprEvaluation(
       Invoke(Literal(obj, clsType), "testFunc", IntegerType, Seq(Literal(1))), 0)
   }
+
+  test("SPARK-35278: static invoke should find method without exact param type match") {
```

Comment:
The JIRA ID looks wrong. Do you want to use SPARK-35288?

Reply:
oops :)